docs/notebooks/at_stackexchange.ipynb

```diff
@@ -0,0 +1,1554 @@
+{
```

Comment:
i am unable to follow this tutorial. please add text and refactor. define all the non-essential code as functions .